components/tidb_query_aggr/src/impl_max_min.rs

```diff
@@ -712,6 +787,77 @@ mod tests {
             col
         }]);
         let logical_rows = vec![3, 2, 6, 5, 1, 7];
+        let expected_res = vec![Some(99), Some(-1i64),];
+        let mut field_type = FieldType::default();
+        let fta = field_type.as_mut_accessor();
+        fta.set_tp(FieldTypeTp::LongLong);
+
+        test_integration_util(field_type.tp(),
+                              field_type.flag(),
+                              &mut columns, &logical_rows, &expected_res);
+    }
+
+    #[test]
+    fn test_aggr_unsigned_signed_int() {
+        // test unsigned bigint
+        let mut columns = LazyBatchColumnVec::from(vec![{
+            let mut col = LazyBatchColumn::decoded_with_capacity_and_tp(0, EvalType::Int);
+            col.mut_decoded().push_int(None);
+            col.mut_decoded().push_int(Some(-1));
+            col.mut_decoded().push_int(Some(2420174916247255494));
+            col.mut_decoded().push_int(Some(3899490809029152765));
+            col
+        }]);
+        let logical_rows = vec![0, 1, 2, 3];
+        let expected_res = vec![Some(-1), Some(2420174916247255494),];
+        let mut field_type = FieldType::default();
+        let fta = field_type.as_mut_accessor();
+        fta.set_tp(FieldTypeTp::LongLong);
+        fta.set_flag(FieldTypeFlag::UNSIGNED);
+
+        test_integration_util(field_type.tp(), field_type.flag(), &mut columns, &logical_rows, &expected_res);
+        
+        // test signed bigint
+        let mut columns = LazyBatchColumnVec::from(vec![{
+            let mut col = LazyBatchColumn::decoded_with_capacity_and_tp(0, EvalType::Int);
+            col.mut_decoded().push_int(None);
+            col.mut_decoded().push_int(Some(-1));
+            col.mut_decoded().push_int(Some(2420174916247255494));
+            col.mut_decoded().push_int(Some(3899490809029152765));
```

Comment:
This test makes no sense. It seems that `3899490809029152765` is still in the representable range of both `i64` and `u64`. You could use some primitives like `std::i64::MAX as u64 + 1` to test this.

Reply:
`-1i64` is `u64::MAX`, so it's lager than 2420174916247255494 and 3899490809029152765.